src/status_im/common/theme/events.cljs

```diff
@@ -10,5 +11,11 @@
 
 (re-frame/reg-event-fx
  :theme/switch
- (fn [{db :db} [theme]]
-   {:db (assoc db :theme theme)}))
+ (fn [{db :db} [{:keys [theme view-id theme-type]}]]
```

Comment:
could you pls add docstring, explaining function and all parameters, thank you

Reply:
Thank you @flexsurfer for reviewing the PR.
Added a docstring, please let me know if needs to be improved further.

Reply:
thank you @Parveshdhull could you please give an example of view-id usage, also why do we have theme and theme-type? probably `theme-type->theme-value` should be done outside this function?